Manatee.Json/Schema/StringFormat.cs

```diff
@@ -75,6 +75,8 @@ public class StringFormat
 		/// </summary>
 		public Func<string, bool> ValidationFunction { get; }
 
+		internal bool IsKnown { get; set; }

```

Comment:
Shouldn't this be default `true` ?
`IsKnown` is now always `false` .. or not ?

Reply:
yeah probably.  good catch.  I'll add it in with my schema draft ~8~ 2019-04 update.